CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+## v6.1.1

```

Comment:
we have to do a major version bump if there is a breaking change in any of the components

Reply:
You are right, there's break change, we need to bump up to 6.2